pmm-app/cypress/support/commands.js

```diff
@@ -0,0 +1,59 @@
+// ***********************************************
+// This example commands.js shows you how to
+// create various custom commands and overwrite
+// existing commands.
+//
+// For more comprehensive examples of custom
+// commands please read more here:
+// https://on.cypress.io/custom-commands
+// ***********************************************
+//
+//
+// -- This is a parent command --
+// Cypress.Commands.add("login", (email, password) => { ... })
+//
+//
+// -- This is a child command --
+// Cypress.Commands.add("drag", { prevSubject: 'element'}, (subject, options) => { ... })
+//
+//
+// -- This is a dual command --
+// Cypress.Commands.add("dismiss", { prevSubject: 'optional'}, (subject, options) => { ... })
+//
+//
+// -- This will overwrite an existing command --
+// Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+import { addMatchImageSnapshotCommand } from 'cypress-image-snapshot/command'
+import { baseUrl, password } from '../../config'
+addMatchImageSnapshotCommand()
+
+Cypress.Commands.add('isVisible', selector => {
+	cy.get(selector).should('be.visible')
+})
+
+Cypress.Commands.add('isNotVisible', selector => {
```

Comment:
This should be the inverse of `isVisible` as the name suggests. Otherwise it'll confuse everyone.
In parallel though, we should probably have a `notPresent` command, which is this one.